src/components/ClientList/ClientList.jsx

```diff
@@ -1,33 +1,23 @@
-import React, { Component } from 'react';
+import React, { PropTypes } from 'react';
 import { connect } from 'react-redux';
 
-export class ClientListComponent extends Component {
-  constructor (props) {
-    super(props);
-
-    this.state = {};
-  }
-
-  render () {
-    return (
-      <div className="row">
-        <ul className="menu" />
-      </div>
-    );
-  }
-}
-
-const mapStateToProps = state => ({
-
-});
-
-const mapDispatchToProps = dispatch => ({
-
-});
-
-const ClientList = connect(
-  mapStateToProps,
-  mapDispatchToProps,
-)(ClientListComponent);
-
-export default ClientList;
+const ClientList = (props) => {
+  const chatItems = props.chats.map(chat => (
+    <li key={chat.id}>{chat.client.first_name}</li>
+  ));
+
+  return (
+    <div className="row">
+      <ul className="menu">{chatItems}</ul>
+    </div>
+  );
+};
+
+ClientList.propTypes = {
+  chats: PropTypes.array.isRequired,
+};
+
+export default connect(
+  () => ({}),
+  () => ({}),
```

Comment:
Could you explain why moving to this formatting is better? I'm not really a fan, specifically in terms of readability,